homeassistant/components/modbus/strings.json

```diff
@@ -68,5 +68,11 @@
         }
       }
     }
+  },
+  "issues": {
+    "deprecated_close_comm_config": {
+      "title": "CLOSE_COMM_ON_ERROR is deprecated",
+      "description": "Please remove parameter \"close_comm_on_error:\" from the modus entry in configuration.yaml."
```

Comment:
Perhaps give a bit more explanation what this means for the user. In the PR description you said sockets will automatically be closed. Is this info relevant for users? Are there situations where people should take more action than removing the key if they have it in there?

Reply:
I added a bit more text, but I do not want to repeat the documentation we already have in place.